### ISSUE TITLE
Aria label-based is too specific with Portugese-based selector

### ISSUE THREAD

**Issue by horatio-sans-serif** [Jhordy9/google-reviews-scraping#1]
> TimeoutError: Waiting for selector `button[aria-label='Classificar avaliações']` failed: Waiting failed: 30000ms exceeded

For me, that would be: `aria-label="Sort reviews"` for example.

Very nice code BTW 👍
